query-engine/connector-test-kit-rs/query-engine-tests/tests/new/interactive_tx.rs

```diff
@@ -71,7 +71,7 @@ mod interactive_tx {
           @r###"{"data":{"createOneTestModel":{"id":1}}}"###
         );
 
-        time::sleep(time::Duration::from_millis(1500)).await;
+        time::sleep(time::Duration::from_millis(10500)).await;
```

Comment:
I'm not sure if this is necessary. But let me re run the tests and see if it makes a difference.